release_util_script_create_gpg.sh

```diff
@@ -23,7 +23,7 @@ gpg --batch --gen-key gen-key-script
 #gpg --quick-gen-key 'Sonny Martin <sonny.martin@diffblue.com>'
 #--------------------------------------------------------------------------------
 ## get key id
-GPG_KEYID=$( gpg --list-keys "Sonny Martin" | head -n1 | grep -Po "(?<=4096R/)[A-Z0-9]+")
+GPG_KEYID=$( gpg --list-keys --with-colons "Sonny Martin" | grep "pub" | head -n1 | cut -d ':' -f5 )
```

Comment:
:grey_question: Is `head -n1` still necessary? I thought `grep` always returned a single line anyway.
Not sure what exactly the output looks like but I'm assuming the `-f5` gets the right entry.

Reply:
Head n1 still necessary in case there are multiple matching keys, gets the most recent. Yes, -f5 does get the right entry as tested on two machines. Next release I might try regenerating the keys again (no harm) just to double check everything is working stably.